sphinxserve/__init__.py

```diff
@@ -114,21 +125,26 @@ def watch(self):
     def render(self):
         '''Render and listen for doc changes (watcher events)
         '''
-        spath = self.c.sphinx_path
         while True:
             self.watch_ev.wait()  # Wait for docs changes
             self.watch_ev.clear()
             with capture_stream() as stdout:
-                build_main(['sphinx-build', spath, spath + '/html'])
+                self.build()
             log.debug(stdout.getvalue())
             self.render_ev.set()
 
+    def build(self):
+        return build_main([
+            'sphinx-build',
+            self.c.sphinx_path,
+            os.path.join(self.c.sphinx_path, self.c.output)
```

Comment:
& because it _is_ a little weird and potentially not commonly known, I feel like I should comment here about how the semantics of `os.path.join` are somewhat surprising in that if your second argument is an absolute path, your first argument is ignored.